src/containers/pages/IRSLite/MapLite/index.tsx

```diff
@@ -0,0 +1,417 @@
+        ]);
+      }
+
+      // get the plan
+      await service(
+        SUPERSET_IRS_LITE_REPORTING_PLANS_SLICE,
+        fetchPlansParams
+      ).then((result: GenericPlan[]) => fetchPlans(result));
+    } catch (e) {
+      displayError(e);
+    } finally {
+      setLoading(false);
+    }
+  }
+
+  useEffect(() => {
+    loadData()
+      .finally(() => setLoading(false))
+      .catch(e => displayError(e));
+  }, []);
+
+  if (!jurisdictionId || !planId) {
+    return <NotFound />;
+  }
+
+  if (loading === true) {
+    return <Loading />;
+  }
+
+  if (!focusArea || !plan || !jurisdiction) {
+    return <ErrorPage errorMessage={AN_ERROR_OCCURRED} />;
+  }
+
+  const baseURL = `${REPORT_IRS_LITE_PLAN_URL}/${plan.plan_id}`;
+  const focusAreaURL = `${baseURL}/${focusArea.jurisdiction_id}`;
+
+  const pageTitle = `${plan.plan_title}: ${focusArea.jurisdiction_name}`;
+
+  const breadcrumbProps = {
+    currentPage: {
+      label: `${focusArea.jurisdiction_name}`,
+      url: `${focusAreaURL}/map`,
+    },
+    pages: [
+      {
+        label: HOME,
+        url: HOME_URL,
+      },
+      {
+        label: IRS_LITE_REPORTING_TITLE,
+        url: REPORT_IRS_LITE_PLAN_URL,
+      },
+      {
+        label: plan.plan_title,
+        url: baseURL,
+      },
+    ],
+  };
+
+  const jurisdictionBreadCrumbs = getJurisdictionBreadcrumbs(focusArea, baseURL);
+
+  const newPages = breadcrumbProps.pages.concat(jurisdictionBreadCrumbs);
+  breadcrumbProps.pages = newPages;
+
+  const indicatorRows = get(IRSIndicatorRows, SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS, null);
+  let sidebarIndicatorRows = null;
+  if (indicatorRows !== null) {
+    sidebarIndicatorRows = getIndicatorRows(indicatorRows, focusArea);
+  }
+
+  const indicatorStops = get(
+    IRSLiteIndicatorStops,
+    SUPERSET_IRS_LITE_REPORTING_INDICATOR_STOPS,
+    defaultIndicatorStop
+  );
+
+  const gisidaWrapperProps = getGisidaWrapperProps(jurisdiction, structures, indicatorStops);
+
+  /**
+   * Create list elements from dictionary
+   * @param {Dictionary} dict
+   */
+  const processListDisplay = (dict: Dictionary | string) => {
+    const createList = (obj: Dictionary) =>
+      Object.entries(obj).map(([key, val]) => (
+        <li key={key} className="indicator-breakdown">
+          {key} - {val} {STRUCTURES}
+        </li>
+      ));
+    return typeof dict === 'string' ? createList(JSON.parse(dict)) : createList(dict);
+  };
+
+  return (
+    <div>
+      <Helmet>
+        <title>{pageTitle}</title>
+      </Helmet>
+      <HeaderBreadcrumb {...breadcrumbProps} />
+      <Row>
+        <Col>
+          <h3 className="mb-3 page-title">{pageTitle}</h3>
+        </Col>
+      </Row>
+      <Row noGutters={true}>
+        <Col xs={9}>
+          {gisidaWrapperProps ? (
+            <div className="map irs-reporting-map">
+              <GisidaWrapper {...gisidaWrapperProps} />
+            </div>
+          ) : (
+            <div>{MAP_LOAD_ERROR}</div>
```

Comment:
We did away with `GisidaWrapper` in favour of [GisidaLite](https://github.com/onaio/reveal-frontend/blob/master/src/components/GisidaLite/index.tsx).

Reply:
This also affects the current `src/containers/pages/IRSLite/MapLite/helpers.ts` code

Reply:
This was converted to use GisdaLite by trying to reuse buildGsLiteLayers() in the FocusInvestigation component. Please see that method to make sure it aligns with your recent refactor.